### PR TITLE
Ship

### PR DESCRIPTION
**Pull request by erin-king**
@earl-stephens Here's the beginning of an epic adventure!

### CODE REVIEW

File: lib/ship.rb [hunk 1]
```diff
@@ -10,4 +10,13 @@ def initialize(name, length)
     @length = length
     @health = length
   end
+
+  def sunk?
```
Comment (earl-stephens) [inline lib/ship.rb]: Don't sink my battleship!

Comment (earl-stephens) [reply lib/ship.rb]: @erin-king

File: lib/ship.rb [hunk 2]
```diff
@@ -19,4 +19,8 @@ def sunk?
     end
   end
 
+  def hit
```
Comment (earl-stephens) [inline lib/ship.rb]: @erin-king that was a hard hit!

**Comment by erin-king** [comment]
Lookin' good!

**Pull request merged**